### ISSUE TITLE
AttributeError: 'GalaxyMNIST' object has no attribute 'data'

### ISSUE THREAD

**Issue by spdc-elm** [mwalmsley/galaxy_mnist#1]
I have ran the python file "load_in_pytorch.py" with the following settings.(I have finished downloaded the file already)
![2022-04-15 21-22-45 的屏幕截图](https://user-images.githubusercontent.com/58126622/163575942-9885aa68-c693-4df7-9dfa-92a4a6bd3eea.png)

![2022-04-15 21-21-09 的屏幕截图](https://user-images.githubusercontent.com/58126622/163575789-c4da67c5-7afb-4fed-b68e-a1672d7f7bab.png)
However, there was a bug.
![2022-04-15 21-23-36 的屏幕截图](https://user-images.githubusercontent.com/58126622/163576018-6e8ee9f0-5b99-42ef-8d69-2f7f3998f494.png)
I've check the file galaxy_mnist.py, and had no idea about why such a bug would happen.
Can you help me? Thanks!

Environment: Ubuntu LTS 20.04, python3.7.11,pytorch 1.11.0

**Comment by mwalmsley** [comment]
Hi there,

Could you let me know your torchvision version, update torchvision, and see
if that helps?

The .data attribute is from the torchvision mnist superclass. It looks from
your first screenshot like it may have downloaded mnist itself, and this
would also likely be due to having an older torchvision version than
galaxyMNIST assumes.

Mike

**Comment by spdc-elm** [comment]
Well, I have checked my torchvision version(0.2.1). I installed it by conda before and I have no idea why it was so old. Now I've downloaded a new one(0.12.0) and run load_in_pytorch.py" again.
![2022-04-29 21-08-44 的屏幕截图](https://user-images.githubusercontent.com/58126622/165950554-4a3749e0-adb2-49d2-bd85-de300b49f06f.png)

Problem was solved. Thanks! 
BTW, I am sorry for typing this comment so late because something important interrupted me.
